netprobify/protocol/target.py

```diff
@@ -55,6 +58,9 @@ def __init__(
         name -- group name/reference
         src_ipv4 -- Bind probes to a specific source IPv4
         src_ipv6 -- Bind probes to a specific source IPv6
+        src_subnet_ipv4 -- Source IPv4 subnet for src ip round robin
+        src_subnet_ipv6 -- Source IPv6 subnet for src ip round robin
+        src_ip_round_robin -- Enable/disable round robin on source IP address to help ECMP
```

Comment:
`src_ip_round_robin` is not useful, since `src_subnet_ipv4` or `src_subnet_ipv6` are defined, source IP round robin should be implicity enabled. You can easily remove this arg and relies only on `src_subnet_ipv4/ipv6` arguments. It will simplify your code as you will have less checks to perform.

Reply:
removed